cognite/client/_api/simulators/simulator_models.py

```diff
@@ -160,3 +165,69 @@ def retrieve_revision(self, id: int | None = None, external_id: str | None = Non
             resource_path="/simulators/models/revisions",
             headers={"cdf-version": "beta"},
         )
+
+    @overload
+    def create_model(self, model: Sequence[SimulatorModel]) -> SimulatorModelList: ...
+
+    @overload
+    def create_model(self, model: SimulatorModel | SimulatorModelWrite) -> SimulatorModelList: ...
+
+    def create_model(
+        self, model: SimulatorModel | SimulatorModelWrite | Sequence[SimulatorModel] | Sequence[SimulatorModelWrite]
+    ) -> SimulatorModel | SimulatorModelList:
+        """`Create one or more simulator models. <https://api-docs.cognite.com/20230101-beta/tag/Simulator-Models>`_
+
+        You can create an arbitrary number of simulator models, and the SDK will split the request into multiple requests.
+
+        Args:
+            model (SimulatorModel | SimulatorModelWrite | Sequence[SimulatorModel] | Sequence[SimulatorModelWrite]): No description.
+
+        Returns:
+            SimulatorModel | SimulatorModelList: Created simulator model(s)
+
+        Examples:
+
+            Create new simulator models::
+
+                >>> from cognite.client import CogniteClient
+                >>> from cognite.client.data_classes import SimulatorModelWrite
+                >>> client = CogniteClient()
+                >>> models = [SimulatorModelWrite(name="model1"), SimulatorModelWrite(name="model2")]
+                >>> res = client.simulators.models.create(models)
+
+        """
+        assert_type(model, "simulator_model", [SimulatorModelCore, Sequence])
+
+        return self._create_multiple(
+            list_cls=SimulatorModelList,
+            resource_cls=SimulatorModel,
+            items=model,
+            input_resource_cls=SimulatorModelWrite,
+            resource_path="/simulators/models",
+            headers={"cdf-version": "beta"},
+        )
+
+    def delete_models(
+        self,
+        id: int | Sequence[int] | None = None,
+        external_id: str | SequenceNotStr[str] | None = None,
```

Comment:
If we support handling more than one, we should call these `ids` and `external_ids`
```suggestion
        ids: int | Sequence[int] | None = None,
        external_ids: str | SequenceNotStr[str] | None = None,
```